src/components/MarcusFEMentor2/IndexPageComponents.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
```

Comment:
try to export 1 component per file; this can be split into 3 and placed under the components folder